### DISCUSSION TITLE
Closing inactive issues pending for >30 days

### DISCUSSION THREAD

**Discussion by gurpreet-wpf** [dotnet/wpf#6573] [Ideas]
Hi All,

In order to organize issues more effectively, we are planning to close the issues where minimal sample repro is not attached and which have been inactive for more than 30 days. 

Any ideas/suggestions are welcome.

Thanks!

**Comment by lindexi** [comment]
@gurpreet-wpf Sorry and I do not think this is a good ideas. Because we can't handle them in time, most Issues is still a unsolved problem, so closing them is not a good way to deal with it. My worry is that closing them will make a lot of developers sad.

**Reply by wstaelens** [reply]
I agree, and it is not always possible to create quickly a repro.
solution with 10+ projects and 10/100-thousand lines of code. Would sometimes take same amount of time to create the repro as for the product. 
(e.g.: the car doesn't drive straight, in order to prove I have to build a new car first,... The central heating in the house doesn't work fine. Sorry, build the house first.)

**Comment by gurpreet-wpf** [comment]
This discussion is closed. #6581 
Thanks for the replies.
